broker/cloud_functions/ps_to_gcs/deploy.sh

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash
+# Deploys or deletes broker Cloud Functions
+# This script will not delete Cloud Functions that are in production
+
+testid="${1:-test}"
+# "False" uses production resources
+# any other string will be appended to the names of all resources
+teardown="${2:-False}"
+# "True" tearsdown/deletes resources, else setup
+survey="${3:-ztf}"
+# name of the survey this broker instance will ingest
+versiontag="${4:-v3_3}"
+PROJECT_ID="${5:-avid-heading-329016}"
+
+#--- GCP resources used in this script
+avro_bucket="${PROJECT_ID}-${survey}_alerts_${versiontag}"
+avro_topic="projects/${PROJECT_ID}/topics/${survey}-alert_avros"
+broker_bucket="${PROJECT_ID}-${survey}-broker_files"
```

Comment:
I don't think this module needs to touch this bucket at all.

Reply:
I've removed any commands that call on broker_bucket